.env

```diff
@@ -3,7 +3,7 @@ FLASK_DEBUG=true
 DEVEL=true
 DATABASE_URL=sqlite:///usn.sqlite3
 SECRET_KEY=insecure_dev_key
-CONTRACTS_API_URL=https://contracts.canonical.com/
+CONTRACTS_API_URL=https://contracts.staging.canonical.com/
```

Comment:
Did you intend to change this?

Reply:
Yes, this is just for anyone on the webteam who QAs it - I'm going to revert this change before merging.